ext/fs/std_fs.rs

```diff
@@ -723,30 +723,34 @@ fn cp(from: &Path, to: &Path) -> FsResult<()> {
     }
   }
 
-  match (fs::metadata(to), fs::symlink_metadata(to)) {
```

Comment:
It's faster because we avoid the second symlink_metadata lookup when the first metadata call is a directory.